app/scripts/actions/index.js

```diff
@@ -65,6 +65,7 @@ export const getCollection = (name, version) => ({
     type: types.COLLECTION,
     method: 'GET',
     id: getCollectionId({name, version}),
+    // path: `collections/${name}/${version}`
```

Comment:
removing this

Reply:
I'm confused: in the collections API, the route is specified as:

`router.get('/:name/:version', get);`

which suggests that the commented out path is correct? Perhaps specifying the collection name and version either via the path or as query params work?

Reply:
I kind of gave up on getting this to work in this iteration. I totally agree that the code I have commented here _should_ work (based on the api documentation and local testing), but for some reason it stalls my dashboard when I use it.

I'm going to spend some time trying to debug that here and let you know what I come up with.

Reply:
https://github.com/nasa/cumulus-dashboard/pull/573/commits/8ec2ae5d957121130c164f456bb7e379f892f6d8